lib/view_component/compiler.rb

```diff
@@ -5,6 +5,14 @@ class Compiler
     # Lock required to be obtained before compiling the component
     attr_reader :__vc_compiler_lock
 
+    # Compiler mode. Can be either:
+    # * blocking (default in Rails development and test mode)
+    # * non_blocking (default in Rails production mode)
```

Comment:
Can you clarify why `blocking` and `non_blocking` are mentioned in the comment? In the latest version, the constants values are `:development` and `:production`.

Reply:
I guess this comment must not have been updated after that change was made.

Reply:
Yea! I have updated the comment, let me know if it's more clear now.

Reply:
Great, thank you!